src/MagnumPlugins/BasisImporter/BasisImporter.h

```diff
@@ -0,0 +1,188 @@
+#ifndef Magnum_Trade_BasisImporter_h
+#define Magnum_Trade_BasisImporter_h
+/*
+This file is part of Magnum.
+
+Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019
+            Vladimír Vondruš <mosra@centrum.cz>
+Copyright © 2019 Jonathan Hale <squareys@googlemail.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNETCION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+*/
```

Comment:
Indentation is off here (it would make it harder for me to do batch copyright updates, then). The comment below seems to be as well.

Actually, the class too -- even though it's excessive, for consistency also the `public:` markers should be indented (so the contents twice).

Reply:
Oh! I was so certain that public would not be indented, that's kind of counter-intuitive to the [overarching goal](https://doc.magnum.graphics/corrade/corrade-coding-style.html#corrade-coding-style-cpp-style)

> The goal is to preserve readability with minimal amount of wasted vertical and horizontal space.

No idea how the license indentation happened 😅

Reply:
I know, it's excessive, but it's done like this everywhere else and I think consistency is what matters. Maybe one day I'll go over all classes and change this to something less excessive, but not yet.